<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProgramDialog</class>
 <widget class="QDialog" name="ProgramDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>306</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">TITLE</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/images/swine32.png</normaloff>:/icons/images/swine32.png</iconset>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <layout class="QGridLayout">
     <item row="3" column="0" colspan="2">
      <widget class="QLabel" name="textLabel2_3">
       <property name="text">
        <string>Working Directory:</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="2" colspan="2">
      <widget class="QLineEdit" name="paramsInput">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="applicationInput"/>
     </item>
     <item row="3" column="3">
      <widget class="QToolButton" name="wdSelectButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icons/images/document-open-folder.png</normaloff>:/icons/images/document-open-folder.png</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="icon">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>image1</normaloff>image1</iconset>
       </property>
       <property name="shortcut">
        <string/>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLineEdit" name="workingDirectoryInput"/>
     </item>
     <item row="1" column="3">
      <widget class="QToolButton" name="exeSelectButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icons/images/document-open.png</normaloff>:/icons/images/document-open.png</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="textLabel1">
       <property name="text">
        <string>Application:</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Name:</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QLabel" name="textLabel2">
       <property name="text">
        <string>Parameters:</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2" colspan="2">
      <widget class="QLineEdit" name="nameInput">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout">
     <item row="1" column="0">
      <widget class="QCheckBox" name="runInTerminalCheckBox">
       <property name="text">
        <string>Run in Terminal</string>
       </property>
       <property name="shortcut">
        <string notr="true"/>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <layout class="QHBoxLayout">
       <item>
        <widget class="QCheckBox" name="desktopCheckBox">
         <property name="text">
          <string>Desktop:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="desktopResolution">
         <property name="editable">
          <bool>true</bool>
         </property>
         <item>
          <property name="text">
           <string notr="true">640x480</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">800x600</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">1024x768</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">1280x1024</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QCheckBox" name="winebootCheckBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Reboot wine</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="logfileCheckBox">
       <property name="text">
        <string>Log to wine.log</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <spacer name="spacer2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>16</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>16</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>16</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>nameInput</tabstop>
  <tabstop>applicationInput</tabstop>
  <tabstop>paramsInput</tabstop>
  <tabstop>icon</tabstop>
  <tabstop>okButton</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>icon</sender>
   <signal>clicked()</signal>
   <receiver>ProgramDialog</receiver>
   <slot>icon_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>ProgramDialog</receiver>
   <slot>cancelButton_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>ProgramDialog</receiver>
   <slot>okButton_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exeSelectButton</sender>
   <signal>clicked()</signal>
   <receiver>ProgramDialog</receiver>
   <slot>exeSelectButton_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>wdSelectButton</sender>
   <signal>clicked()</signal>
   <receiver>ProgramDialog</receiver>
   <slot>wdSelectButton_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
